Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicTreadmillData.swift

```diff
@@ -200,10 +200,10 @@ final public class CharacteristicTreadmillData: Characteristic {
         
         if flags.contains(.elevationGainPresent) {
             let pValue = Double(decoder.decodeUInt16(data))
-            pElevaionGain = Measurement(value: pValue, unit: UnitLength.meters)
+            pElevaionGain = Measurement(value: pValue, unit: UnitLength.decimeters)
```

Comment:
I think this actually still should be meters.. relooking at the docs it is in units of meters with resolution of 0.1

Reply:
I am onboard with that, I will update the pull request to use .resolution